health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -284,20 +288,45 @@ const CampaignBoundary = ({ customProps }) => {
     setBoundaryStatus(updatedBoundaryStatus);
   }, [boundaryHierarchy]); // Only re-run when boundaryHierarchy changes
 
+  const editHandler = () => {
+    const urlParams = Digit.Hooks.useQueryParams();
+    urlParams.key = '3';
+    const updatedUrl = `${window.location.pathname}?${new URLSearchParams(urlParams).toString()}`;
+    history.push(updatedUrl);
+  }
+  const isEditable=setupCompleted==='true'? false:true;
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)